### PR TITLE
Add  sp60

### PR DESCRIPTION
**Pull request by Hiiwwy**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hmc_keyboards/sp60/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS)
+		};
+
+
+
+
+
+
+
+
```
Comment (zvecr) [inline keyboards/hmc_keyboards/sp60/keymaps/via/keymap.c:91]:
```suggestion
		KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_LALT, KC_LGUI, KC_MENU, KC_LCTL, KC_LEFT, KC_DOWN, KC_RGHT)
		};
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
No point requesting review, when multiple suggestions have not been completed (or have been done incorrectly).

File: keyboards/hmc_keyboards/sp60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+SLEEP_LED_ENABLE    = no       # Breathing sleep LED during USB suspend
+OLED_DRIVER_ENABLE  = no
+ALLOW_WARNINGS      = yes
+LG_Indicator_Enable = yes
+RGB_LG              = yes
+#WS2812_DRIVER       = pwm
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+# OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/hmc_keyboards/sp60/rules.mk]: Stop marking this as resolved, when you have not implemented the changes.

File: keyboards/hmc_keyboards/sp60/keymaps/via/keymap.c (hunk 1 above)
Comment (zvecr) [reply keyboards/hmc_keyboards/sp60/keymaps/via/keymap.c]: This has not been implemented per the suggestion.

**Comment by zvecr** [comment]
Closing PR as suggestions are continually being ignored.

**Comment by Hiiwwy** [comment]
I didn't ignore the suggestion, probably because I didn't understand it

**Comment by Hiiwwy** [comment]
> 关闭 PR 作为建议不断被忽略。

I didn't ignore the suggestion, probably because I didn't understand it

**Comment by Hiiwwy** [comment]
> 关闭 PR 作为建议不断被忽略。

I didn't ignore the suggestion, probably because I didn't understand it

**Comment by zvecr** [comment]
Intentional or not, you are ignoring them. If you dont understand, ask. 

The logic of closing a suggestion multiple times, without any action, is ridiculous.

**Pull request closed without merging**